anet-dictionary.yml

```diff
@@ -3,6 +3,7 @@ CONNECTION_ERROR_MSG: Connection to the server is lost
 VERSION_CHANGED_MSG: "There is a new version of ANET, click here to load the new version (CAUTION: save any unsaved changes first)"
 SUPPORT_EMAIL_ADDR: support@example.com
 
+regularUsersCanCreateLocations: false
```

Comment:
We should probably set this to `true` in our development dictionaries so we use the functionality ourselves.
And then also change the server-side test (i.e. reverse the conditions).